src/plugins/vis_type_vega/public/vega_fn.ts

```diff
@@ -51,7 +47,7 @@ export type VegaExpressionFunctionDefinition = ExpressionFunctionDefinition<
   Input,
   Arguments,
   Output,
-  ExecutionContext<VegaInspectorAdapters>
+  ExecutionContext<VegaInspectorAdapters, ExecutionContextSearch>
```

Comment:
For plugins registering expression functions which rely on the search execution context and want to maintain the same level of type safety as we had previously, this is how they can go about achieving it.

For this PR I did not comb through every registered expression function to make this change; rather, I only looked for places where a specific `ExecutionContext` was already being included.